.github/workflows/trunk.yml

```diff
@@ -94,25 +94,44 @@ jobs:
       build-environment: linux-xenial-cuda11.3-py3.7-gcc7-no-ops
       docker-image-name: pytorch-linux-xenial-cuda11.3-cudnn8-py3-gcc7
 
-  linux-bionic-rocm4_5-py3_7-distributed-build:
+  linux-bionic-rocm5_0-py3_7-distributed-build:
     name: linux-bionic-rocm5.0-py3.7-distributed
     uses: ./.github/workflows/_linux-build.yml
     with:
       build-environment: linux-bionic-rocm5.0-py3.7
       docker-image-name: pytorch-linux-bionic-rocm5.0-py3.7
 
-  linux-bionic-rocm4_5-py3_7-distributed-test:
+  linux-bionic-rocm5_0-py3_7-distributed-test:
     name: linux-bionic-rocm5.0-py3.7-distributed
     uses: ./.github/workflows/_rocm-test.yml
-    needs: linux-bionic-rocm4_5-py3_7-distributed-build
+    needs: linux-bionic-rocm5_0-py3_7-distributed-build
     with:
       build-environment: linux-bionic-rocm5.0-py3.7
-      docker-image: ${{ needs.linux-bionic-rocm4_5-py3_7-distributed-build.outputs.docker-image }}
+      docker-image: ${{ needs.linux-bionic-rocm5_0-py3_7-distributed-build.outputs.docker-image }}
       test-matrix: |
         { include: [
           { config: "distributed", shard: 1, num_shards: 1, runner: "linux.rocm.gpu" },
         ]}
 
+  linux-bionic-rocm5_0-py3_7-slow-build:
```

Comment:
Should a slow test be added to periodic rather than having it be on trunk?

Trying to be mindful of our limited resources, since adding any long running workflows could tie up resources for a long time

Reply:
Agree.  Will move to periodic.  I wanted to see one CI signal to know how slow the slow jobs take before I decided trunk vs periodic, and clearly it needs to be periodic.

Is there a ciflow label to trigger the periodic jobs?  I'm keeping this PR in draft mode until I get a good signal.

Reply:
Done.  Moved to periodic.  Added periodic label to PR to trigger tests.